machine/wnc_x86_64/wnc.patch/installer/x86_64/grub.d/50_onie_grub

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
```

Comment:
In this file, I just want to turn on GRUB_CMDLINE_LINUX and RUB_SERIAL_COMMAND, write parameters in it which I need, like --speed, --unit. Maybe this should modify at my local side?